os/arch/xtensa/Kconfig

```diff
@@ -0,0 +1,86 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+if ARCH_XTENSA
+
+choice
+	prompt "XTENSA architecture selection"
+	default ARCH_CHIP_ESP32
+
+config ARCH_CHIP_ESP32
+	bool "Expressif ESP32"
+	select ARCH_FAMILY_LX6
+	select XTENSA_HAVE_INTERRUPTS
+	select ARCH_HAVE_MULTICPU
+	select ARCH_TOOLCHAIN_GNU
+	---help---
+		The ESP32 is a dual-core system from Expressif with two Harvard
+		architecture Xtensa LX6 CPUs. All embedded memory, external memory
+		and peripherals are located on the data bus and/or the instruction
+		bus of these CPUs. With some minor exceptions, the address mapping
+		of two CPUs is symmetric, meaning they use the same addresses to
+		access the same memory.
+
+		The two CPUs are named "PRO_CPU" and "APP_CPU" (for "protocol" and
+		"application"), however for most purposes the two CPUs are
+		interchangeable.
+
+endchoice # XTENSA chip selection
+
+config ARCH_FAMILY_LX6
+	bool
+	default n
+	---help---
+		Cadence® Tensilica® Xtensa® LX6 data plane processing unit (DPU).
+		The LX6 is a configurable and extensible processor core.
+
+config ARCH_CHIP
+	string
+	default "esp32"		if ARCH_CHIP_ESP32
+
+config XTENSA_CP_LAZY
+	bool "Lazy co-processor state restoration"
+	default n
+	depends on EXPERIMENTAL
+	---help---
+		NuttX logic saves and restores the co-processor enabled (CPENABLE)
```

Comment:
Our project is NOT Nuttx. Please check all of `Nuttx` written.

Reply:
Fixed, Thanks.